package.json

```diff
@@ -39,24 +39,24 @@
     "awaitqueue": "^2.3.3",
     "bowser": "^2.11.0",
     "debug": "^4.3.1",
-    "events": "^3.2.0",
+    "events": "^3.3.0",
     "h264-profile-level-id": "^1.0.1",
     "sdp-transform": "^2.14.1",
     "supports-color": "^8.1.1"
   },
   "devDependencies": {
     "@types/sdp-transform": "^2.4.4",
-    "@types/uuid": "^3.4.9",
-    "@typescript-eslint/eslint-plugin": "^4.14.2",
-    "@typescript-eslint/parser": "^4.14.2",
-    "eslint": "^7.19.0",
-    "eslint-plugin-jest": "^24.1.3",
+    "@types/uuid": "^8.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.16.1",
+    "@typescript-eslint/parser": "^4.16.1",
+    "eslint": "^7.21.0",
+    "eslint-plugin-jest": "^24.1.5",
     "fake-mediastreamtrack": "^1.1.6",
     "jest": "^26.6.3",
     "jest-tobetype": "^1.2.3",
     "open-cli": "^6.0.1",
     "tsc-watch": "^4.2.9",
-    "typescript": "~3.7.5",
-    "uuid": "^3.4.0"
+    "typescript": "~4.2.2",
+    "uuid": "^8.3.2"
```

Comment:
Please let's not update deps (unrelated to this PR).

- Also AFAIR `uuid` 8.X.X has some issues in React-Native.
- Also, there are reasons for we to stay in typescript 3.X.X (otherwise Angular users will complain).

I'll take the valid changes and apply them into my ongoing PR.

Reply:
Sure, will reduce to the basic requirements.

Reply:
AFAIS `export type` does not work with typescript 3.X.X.

Reply:
Should work since TS 3.8 https://www.typescriptlang.org/docs/handbook/modules.html#importing-types

Reply:
Great